website/photos/api/v2/views.py

```diff
@@ -51,6 +55,35 @@ def get_queryset(self):
         )
 
 
+class LikedPhotosListView(ListAPIView):
+    """Returns the details the liked album."""
+
+    serializer_class = PhotoListSerializer
+    permission_classes = [
+        IsAuthenticatedOrTokenHasScope,
+    ]
+    required_scopes = ["photos:read"]
+
+    def get(self, request, *args, **kwargs):
+        if not self.request.member:
```

Comment:
Don't we have a permission class for this?

Reply:
Looks like we actually don't. Currently all API endpoint use `IsAuthenticatedOrTokenHasScope`. I think we should probably change that though.
Still, in this specific case a permission class (only) doesn't really make sense, as the check is a technical requirement, not really about permissions.